src/sympc/tensor/mpc_tensor.py

```diff
@@ -206,6 +199,28 @@ def __init__(
         self.grad_fn = None
         self.parents: List["MPCTensor"] = []
 
+    @staticmethod
+    def distribute_shares(shares: List[ShareTensor], session: Session):
```

Comment:
Q: George , In your Previous [PR](https://github.com/OpenMined/SyMPC/pull/178),share distribution was made at Protocol level.Now it is modified again to distribute at MPC Tensor?

Reply:
I will replace this.